plutus-core-spec/plutus-core-specification.bib

```diff
@@ -1,3 +1,64 @@
+@InProceedings{unravelling-recursion,
```

Comment:
I wonder if we could have a shared bib file for all our publications... they have a lot in common!